packages/nextjs/components/scaffold-eth/Address.tsx

```diff
@@ -16,6 +16,8 @@ type AddressProps = {
   disableAddressLink?: boolean;
   format?: "short" | "long";
   size?: "xs" | "sm" | "base" | "lg" | "xl" | "2xl" | "3xl";
+  textClasses?: string;
+  cardClasses?: string;
```

Comment:
I think it's better to name props `textClassName` and `cardClassName` similar to base `className` prop